staging/src/k8s.io/apiextensions-apiserver/test/integration/versioning_test.go

```diff
@@ -88,11 +88,11 @@ func TestInternalVersionIsHandlerVersion(t *testing.T) {
 	{
 		t.Logf("patch of handler version v1beta1 (non-storage version) should succeed")
 		i := 0
-		err = wait.PollImmediate(time.Millisecond*100, wait.ForeverTestTimeout, func() (bool, error) {
+		err = wait.PollUntilContextTimeout(context.Background(), time.Millisecond*100, wait.ForeverTestTimeout, true, func(ctx context.Context) (done bool, err error) {
 			patch := []byte(fmt.Sprintf(`{"i": %d}`, i))
 			i++
 
-			_, err := noxuNamespacedResourceClientV1beta1.Patch(context.TODO(), "foo", types.MergePatchType, patch, metav1.PatchOptions{})
+			_, err = noxuNamespacedResourceClientV1beta1.Patch(ctx, "foo", types.MergePatchType, patch, metav1.PatchOptions{})
```

Comment:
Please leave the `:=`.

Reply:
Fixed it https://github.com/kubernetes/kubernetes/commit/ed7f7ce92b06b2452a3d879b4c70d34f5d31ff23